### ISSUE TITLE
Cyclepath is never used even tho OrganicMaps sees it as Cyclepath

### ISSUE THREAD

**Issue by carnager** [organicmaps/organicmaps#9844]
### Issue
In my Area (Wuppertal, Germany) there is an old railway that is now the biggest cyclepath of the city. Sadly Organicmaps refuses to use it for routing even tho when clicking it, it's clearly labeled as "Cyclepath"

### Location of the Cycling Path
The path in question is here:
https://osm.org/go/0GJcU2sT-?m

### Display of the Path in OM
Organicmaps shows the following for it:
<img src="https://github.com/user-attachments/assets/be1336a0-19bd-47b9-a10c-3521115c408a" width="200" />

### Reproduction
Try a route anywhere, where the obvious best route would be the trail .

### Information

OS: Android 15
OM Version: 2024.11.27-12-Fdroid

**Comment by map-per** [comment]
Bicycle routing on that path works on my phone. Are you using the latest map data?

**Comment by carnager** [comment]
Yes I am. it always routes me over "Berliner Straße" which is the biggest road in town with lots of cars. 

try "Rathenaustraße, Wuppertal" to "e.g. "Ehrenhainstraße" which was my original query. both are connected 1:1 via the bike trail.

**Comment by map-per** [comment]
Ok, so Organic Maps doesn't prefer the cycleway for e.g. [this route](https://www.openstreetmap.org/directions?engine=graphhopper_bicycle&route=51.27761%2C7.21228%3B51.23971%2C7.07992#map=13/51.26014/7.15691). Looks like OM chooses the route via the street because the cycleway is assumed to have more elevation diff (223 up, 242 down vs. 97 up, 116 down). The elevation is probably not correct though, a former railway should have very low elevation diff.

**Comment by map-per** [comment]
@pastk This is an interesting test case for improving the cycle routing

**Comment by pastk** [comment]
> the cycleway is assumed to have more elevation diff (223 up, 242 down vs. 97 up, 116 down). The elevation is probably not correct though, a former railway should have very low elevation diff.

Yeap it seems to be the main reason
- related to https://github.com/organicmaps/organicmaps/issues/5123

(ref: according to GraphHopper its `Ascend: 163m. Descend: 173m.` via the cycleway)

The cycleway route is also noticeably shorter 10940m vs 12280m via the road.

Also where OM routes via roads they usually have `hwtag-yesbicycle` which is overemphasized often #9593.
